VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -544,5 +547,11 @@ private async Task<string> GetParticipantIdByUsernameAsync(Guid conferenceId, st
                 return particiantId;
             }
         }
+
+        public async Task UpdateHearingLayout(Guid conferenceId, HearingLayout newLayout)
+        {
+            var updatedById = Guid.Parse(await GetParticipantIdByUsernameAsync(conferenceId, Context.User.Identity.Name));
```

Comment:
Should we add a logger here?

Reply:
I am moving this as per Farron's comment. I will also add additional logging.